test/functionalTest/cases/0000_ngsi9_subscriptions/notify_format_in_ngsi9_subscriptions.test

```diff
@@ -685,7 +685,7 @@ Content-Type: application/json; charset=utf-8
             "isDomain" : "false"
```

Comment:
There are no match cases with 99** , 9802,9803,9804,9805,9999 and 5683.

Only cases for 9801.

Fixed in 0381eb2